OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -99,6 +99,16 @@ public class OSRMInstructionFormatter: Formatter {
         return instructions["constants"] as! [String: Any]
     }
     
+    /**
+     Returns a format string with the given name.
+     
+     - returns: A format string suitable for `String.replacingTokens(using:)`.
+     */
+    public func phrase(named name: PhraseName) -> String {
+        let phrases = instructions["phrase"] as! [String: String]
+        return phrases["\(name)"]!
```

Comment:
Does it make sense to make this return optional?

Reply:
`PhraseName` is a limited set of values, so if a valid `PhraseName` is passed in, I think the return value should be required.